packages/patternfly-4/content/design-guidelines/usage-and-behavior/forms/design.md

```diff
@@ -10,110 +10,148 @@ Most forms will contain the following basic elements:
 *Form elements*
 ![Form elements](img/form-elements.png)
 
-1. [Labels](/design-guidelines/usage-and-behavior/data-input) - Field labels can be aligned to the left or top of the field depending on the layout of your page and the amount of space you have to work with. Learn more about field labels in the [text and data input design guidelines](/design-guidelines/usage-and-behavior/data-input).
-
-  *Aligned to the top*
+1. [Labels](#labeling) - Field labels can be aligned to the left or top of the field depending on the layout of your page and the amount of space you have to work with.
 
   ![Aligned to the top](img/top-aligned.png)
```

Comment:
I wonder whether these two images should be moved to the section below where you talk about labeling. As I read this I expected to find them there.